lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -4609,6 +4609,36 @@ LogicalResult ConvertAtenOp<AtenSqrtOp>::matchAndRewrite(
   return success();
 }
 
+template <typename AtenOpT>
+class ConvertAtenOpToTosaCustomOp : public OpConversionPattern<AtenOpT> {
+public:
+  using OpConversionPattern<AtenOpT>::OpConversionPattern;
+  using OpAdaptor = typename AtenOpT::Adaptor;
+
+  LogicalResult
+  matchAndRewrite(AtenOpT op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+
+    // Remove torch.aten. prefix from operation name.
+    StringLiteral torchOpName = AtenOpT::getOperationName();
+    auto opName = torchOpName.drop_front(torchOpName.find_last_of(".") + 1);
```

Comment:
Some ops are called `aten.bla.Scalar`, which would become `Scalar` here?
Maybe instead of being to fancy, we can just keep the whole torchOpName (for now?).

Reply:
Or, maybe better, pass in the identifier of the custom op as second template parameter (alternative: constructor argument).

Reply:
IMO the main thing is to keep all the information needed to know what the operation is, so stripping information doesn't seem useful.

But if you know that the converted operands and result types will be exactly what is needed to call an operation from a more standard dialect, like `math.aten2`, then it would make sense to replace it with that. I don't know the details, but believe the conversion to that dialect could be trivially implemented.

Reply:
Thanks.
For now, I'll have a constructor argument.